### ISSUE TITLE
Multi Label Classification ?

### ISSUE THREAD

**Issue by benihime91** [artemmavrin/focal-loss#13]
Is `BinaryFocalLoss` applicable for MultiLabel Classification Problems ?

**Comment by artemmavrin** [comment]
Hi @benihime91 can you give an example of what you mean? Similar to the ordinary sigmoid cross-entropy loss, `BinaryFocalLoss` is used to assign one of two exclusive labels to data.

**Comment by benihime91** [comment]
I am taking about something similar to [this](https://www.pyimagesearch.com/2018/05/07/multi-label-classification-with-keras/). 

I am looking for the case where my targets for a single input can be  : `[1, 0, 1, 0]` instead of `[1, 0, 0, 0]`

**Comment by artemmavrin** [comment]
Hi @benihime91 I'm sorry I missed your follow-up almost a month ago. You can use `BinaryFocalLoss` for this, if I'm understanding your application correctly.
